pkgs/development/python-modules/blocksat-cli/default.nix

```diff
@@ -35,11 +35,15 @@ buildPythonPackage rec {
   pytestFlagsArray = [
     # disable tests which require being connected to the satellite
     "--ignore=blocksatcli/test_satip.py"
+    "--ignore=blocksatcli/api/test_listen.py"
+    "--ignore=blocksatcli/api/test_msg.py"
     "--ignore=blocksatcli/api/test_net.py"
     # disable tests which require being online
     "--ignore=blocksatcli/api/test_order.py"
   ];
```

Comment:
We now have a fancy option called disabledTestPaths which also checks if the files still exists. If you get a minute it would be nice if we could convert this.

Reply:
Thanks, PR in https://github.com/NixOS/nixpkgs/pull/146369